### ISSUE TITLE
Gerald needs a new tube

### ISSUE THREAD

**Issue by DefProc** [DoESLiverpool/somebody-should#266]
I've just been testing Gerald to find out the cause of the variable cutting I've been getting today, when the output power faded out to almost nothing. This is almost certainly a tube that's gone, the current tube was last replaced before the one we just replaced in Sophia. 

We'll need to order and fit a new tube.

**Comment by DefProc** [comment]
@DoESsean has ordered us a tube.

**Comment by DefProc** [comment]
The new tube is fitted although it was slightly larger diameter, so I had to [make mounts](https://github.com/DoESLiverpool/somewhere-safe/commit/e9b4daff12e1f5843768808fe65b28a668326639) to fit it. In the end, the alignment wasn't too far off (hitting all three mirrors and marking on the bed), but it still showed:

 * reduced power,
 * having better cuts moving in the x-axis than in the y-axis,
 * fading out over a few cuts.

On the plus side, this did suggest that the outgoing tube was still fine (and could still be used again) so it's wrapped up in the corner of the workshop and we have a spare. 

I did then have a thought that it might be hitting the side of the cone (*a la* #260) and removed it to check, only to find that it could suddenly cut like a demon! Although this is different behaviour than we've seen before (ghosted, out of focus, crescent shaped second cut mark), my suspicion is that the beam has been elongated in the x-axis slightly (causing better cuts in this direction), and that the laser hitting the aluminium part then caused the fadeout as it expanded with the heat from the laser. 

As it stands, Gerald is still marked broken because it's not correctly aligned, although as long as it's checked that the alignment is accurate enough in both the top-left and bottom-right corners, it could — in a pinch — be used with the cone removed and the bare air pipe fixed pointing downwards (as on Sophia). If the alignment is out though, the accuracy of size of cut parts may be greater than the normal ±0.25mm range.
